calicoctl/calico_ctl/node.py

```diff
@@ -82,11 +83,16 @@
                             will be launched via the calico-node container,
                             whereas a value of "none" will not launch them at
                             all. [default: docker]
+  --backend=<BACKEND>       Specify which networking backend to use.
+                            Choices are "bird", "gobgp" or "none".
+                            When set to "none", Calico node run in policy
+                            only mode. [default: bird]
```

Comment:
@ishidawataru Did you modify this docopt string?

`[default: bird]` needs to be removed.

And the `[--backend=<BACKEND>]` needs to be changed back to `[--backend=(gobgp|none|bird)]`.

This is important so that the `calicoctl/node.py` code can properly detect if the user is using the old `CALICO_NETWORKING` variable, or if they're using the new `--backend` flag. If `[default: bird]` is there, its impossible for the python code to tell if the user specified `--backend` or if they're using the old CALICO_NETWORKING env.